language-service/src/services/yamlCompletion.ts

```diff
@@ -200,7 +214,7 @@ export class YAMLCompletion {
                         insertText: this.getInsertTextForProperty(currentWord, null, false, separatorAfter),
                         insertTextFormat: InsertTextFormat.Snippet,
                         documentation: ''
-                    });
+                    }, false);
```

Comment:
Since this is a property suggestion, why not to check ignore case?

Reply:
This block is run when there is no schema so we should use the default case-sensitive behavior.